package.json

```diff
@@ -1,17 +1,22 @@
 {
   "version": "1.0.0",
   "scripts": {
-    "test": "node_modules/.bin/lerna run test",
-    "lint": "node_modules/.bin/lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
-    "doc": "node_modules/.bin/lerna run doc",
-    "audit": "node_modules/.bin/lerna run audit",
-    "build": "node_modules/.bin/lerna run build",
-    "postinstall": "node_modules/.bin/lerna bootstrap --no-ci && node_modules/.bin/lerna run build"
+    "test": "lerna run test",
+    "lint": "lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
+    "doc": "lerna run doc",
+    "audit": "lerna run audit",
+    "build": "lerna run build",
+    "postinstall": "node_modules/.bin/lerna bootstrap --no-ci && node_modules/.bin/lerna run build",
```

Comment:
The `node_modules/.bin/lerna run build` part can be removed if you rename build scripts to prepare scripts for all packages. It will be more conceptional and during the release the prepare script is executed automatically without having to call it explicitly.

Reply:
Also, since you are doing `node_modules/.bin/lerna` -> `lerna`, you missed this spot.

Reply:
"build" scripts renamed to "prepare" scripts.